pyccel/codegen/printing/ccode.py

```diff
@@ -250,18 +254,38 @@ def _print_Complex(self, expr):
                         PyccelMul(expr.imag, ImaginaryUnit()))))
 
     def _print_PythonComplex(self, expr):
-        self._additional_imports.add("complex.h")
+        self._additional_imports.add("complex")
         return self._print(PyccelAssociativeParenthesis(PyccelAdd(expr.real_part,
                         PyccelMul(expr.imag_part, ImaginaryUnit()))))
 
     def _print_ImaginaryUnit(self, expr):
         return '_Complex_I'
 
+    def _print_ModuleHeader(self, expr):
+        name = expr.module.name
+        # TODO: Add classes and interfaces
+        funcs = ';\n\n'.join(self.function_signature(f) for f in expr.module.funcs)
```

Comment:
I think here if the module contains only one function, the `';\n\n'` will not be added.

Reply:
Yes, I added the semi colon for the final function in the assembly in the return. But I think you're right that it's not very clear that way. I will change it